package.json

```diff
@@ -9,6 +9,9 @@
     "url": "https://github.com/Automattic/wp-calypso.git"
   },
   "main": "index.js",
+  "bin": {
+    "wp-calypso-sdk": "./bin/create-scripts/block.js"
+  },
```

Comment:
also, TIL how to make `bin` scripts in `npm` and it's much easier than I anticipated! thanks for that @simison